### PR TITLE
Shadow glove calibration: fixing autoconfigure of fingertip mapping

### PR DESCRIPTION
**Pull request by ethanfowler**
…caling, YAML loader warnings

## Proposed changes

Please explain the changes you made. Add pictures or videos to explain them better if appropriate.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [x] Manually tested on hardware (if hardware specific or related).

## Documentation

- [x] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

**Comment by ethanfowler** [comment]
This needs testing in HW @georgiablanco !

**Review by bdenoun — COMMENTED** [review]
IF we try to apply the boyscout rule, would it be possible to have the whole file type hinted? :)

File: src/polhemus_ros_driver/knuckle_position_calibration_action_server.py [hunk 1]
```diff
@@ -289,7 +290,7 @@ def _get_connected_glove_prefixes():
         tf_buffer = tf2_ros.Buffer()
         tf2_ros.TransformListener(tf_buffer)
         rospy.sleep(5)
-        connected_glove_sides = []
+        connected_glove_sides: "list[str]" = []
```
Comment (bdenoun) [inline src/polhemus_ros_driver/knuckle_position_calibration_action_server.py]: Why is that between quotes?

Comment (georgiablanco) [reply src/polhemus_ros_driver/knuckle_position_calibration_action_server.py]: fixed in https://github.com/shadow-robot/polhemus_ros_driver/pull/46/commits/f7c5c1e4c8f5b7457c74ba456e49c55c3c8f0e99

**Review by bdenoun — APPROVED** [review]

**Review by rnzenha-s — APPROVED** [review]

**Pull request merged**